### ISSUE TITLE
Doesn't scan CIFS/Samba network directories

### ISSUE THREAD

**Issue by msx** [gnumdk/lollypop#267]
Hi gnumdk,

I'm not really sure if this is an issue from Lollypop itself[0] so please have a little patience here :)
At the moment Lollypop is unable to add music files that are on mounted CIFS directories.
I tried:
1. Adding the directory as presented by Nautilus when connecting to a network shared folder;
2. Manually mounting the endpoint to a local directory (/media/something) then adding it to Lollypop.
In both cases files can be accessed without a problem by other applications like Clementine or Tomahawk.

[0] As I'm don't know about Lollypop internal I can't really say if this is a problem of the application itself or from the backend it uses. For example, I know that Tracker is used to search for files across users' directories and present them to GNOME's applications so the user has an 'automated' browsing experience. If Lollypop does use Tracker as its backbone then there may lie the blocker.

Cheers.

**Comment by gnumdk** [comment]
Here it works...

**Comment by gnumdk** [comment]
lollypop -d output?

**Comment by eonpatapon** [comment]
No problem here too. I use systemd automount to mount my cifs shares on the system.

Lollypop doesn't use tracker (thanks god!).

**Comment by gnumdk** [comment]
Reopen if you have more informations.

**Comment by gavsiu** [comment]
I have networked CIFS shares mounted in Thunar which uses gvfs-mount.

The mounted shares do not show up in the browse window. I managed to select the folder through the Recent tab, but Lollypop does nothing even when I click Update music.

In Rhythmbox, the network share shows up in the browse window. I can set it as the default folder and it imports the music into the database. However, I really dislike the display options.

Lollypop seems to be what I'm looking for, but I can't get it to work.
